azkaban-common/src/test/java/azkaban/metric/MetricManagerTest.java

```diff
@@ -81,12 +81,12 @@ public void managerEmitterMaintenanceTest() {
   @Test
   public void managerEmitterHandlingTest() throws Exception {
     this.emitter.purgeAllData();
-    final Date from = DateTime.now().minusMinutes(1).toDate();
+    final Date from = DateTime.now().minusMinutes(10).toDate();
     this.metric.notifyManager();
 
     this.emitterWrapper.countDownLatch.await(10L, TimeUnit.SECONDS);
 
-    final Date to = DateTime.now().plusMinutes(1).toDate();
+    final Date to = DateTime.now().plusMinutes(10).toDate();
```

Comment:
Do you know somehow that this was the culprit? Even 1 minute is a lot, so I would expect the bug to lie somewhere else.

Reply:
Sure, that is an interesting question. :) I posed that to myself and the result you can see in the updated PR. :)